src/frontends/paddle/tests/test_models/gen_scripts/generate_unique.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# unique paddle model generator
+#
+import numpy as np
+from save_model import saveModel
+import paddle
+import sys
+
+data_type = "float32"
+
+
+def unique(name: str, x, **op_args):
+    paddle.enable_static()
+
+    with paddle.static.program_guard(paddle.static.Program(), paddle.static.Program()):
+        node_x = paddle.static.data(name="x", shape=x.shape, dtype=data_type)
+
+        unique_outs = paddle.unique(node_x, **op_args)
+        if isinstance(unique_outs, tuple):
+            outputs = []
+            for out in unique_outs:
+                if out is not None:
+                    if out.dtype == paddle.int64 or out.dtype == paddle.int32:
+                        out = paddle.cast(out, "float32")
```

Comment:
I can't figure out why `cast` operation is needed. In my understanding, we should keep the original `unique` output for verification.

Reply:
AFAIK, the openvino test [compare_results](https://github.com/openvinotoolkit/openvino/blob/master/src/tests/test_utils/common_test_utils/src/test_case.cpp#L169-L179) only support fp32 & int32, so for the int64 index we should convert to float32/int32

Reply:
@AndPuQing Thanks for your very detailed analysis. Your finding makes sense.
Maybe the test case framework is not perfect enough to call. Don't worry at all. If you think it is not reasonable, just pull PR to fix or optimize it, which is much appreciated. We should NOT skip it using a work-around.
If you have no time for this case, let me know.

Reply:
Sorry, it seems I may not have the time to rectify the test case framework.